tests/phpunit/unit/api/image/includes/class-image-test-case.php

```diff
@@ -133,6 +122,8 @@ private function set_up_virtual_filesystem() {
 
 		// Set up the "beans" directory's virtual filesystem.
 		$this->mock_filesystem = vfsStream::setup( 'uploads', 0755, $structure );
+		$this->images_dir      = vfsStream::url( 'uploads/beans/images' );
+		$this->images_url      = 'http:://example.com/uploads/beans/images/';
```

Comment:
Should `http::`  be `http:`  or `https:`?

Reply:
Good catch, @christophherr.